bin/cron/create_rollup_tables

```diff
@@ -143,11 +143,6 @@ def main
     end
   end
 
-  # Verify that the data size is sensible.
-  raise "contained_levels too big: #{contained_levels.size} rows exceeds limit of 10000" if contained_levels.size > 10_000
-  raise "contained_level_answers too big: #{contained_levels_answers.size} rows exceeds limit of 100000" if contained_level_answers.size > 100_000
```

Comment:
It looks like the error was being raised on line 147. I wonder if we can keep these other warnings in for now if we're concerned?

In particular, I think we hold all levels in memory during seeding (ie [here](https://github.com/code-dot-org/code-dot-org/blob/d2dffe78c33a1c969547fcb6fea0ace61726783d/dashboard/lib/tasks/seed.rake#L236) for DSLDefined levels)) so I think we'd start to see problems earlier than this if we had too many levels. Too many level sources is a different issue though as they're user generated